yarn-project/aztec.js/src/wallet/base_wallet.ts

```diff
@@ -54,17 +53,10 @@ import type {
  * Options to configure fee payment for a transaction
  */
 export type FeeOptions = {
-  /** Whether to use an specific payment method */
-  paymentMethod?: FeePaymentMethod;
-  /**
-   * Whether the account contract assembling the tx should set itself as fee payer or not
-   */
-  isFeePayer: boolean;
-  /**
-   * In case no external fee payment method is provided, whether the account
-   * contract executing the transaction should end the setup phase
-   */
-  endSetup: boolean;
+  /** Wallet provided fee payment method, in case the tx doesn't include one */
```

Comment:
```suggestion
  /**
   * A wallet-provided fallback fee payment method that is used only if the transaction that is being constructed
   * doesn't already include one
   */
```
I found the original a bit confusing because the term "transaction" in blockchains in generally refers to a signed immutable object. Hence the wallet having the ability to modify it is strange. Pardon my autism